bin/critterget.py

```diff
@@ -781,6 +770,34 @@ def getCrashCounts(appId,appName):
     return
 
 
+def setDataRegion(sessionKey):
+    """
+    :param sessionKey: A string generated by Splunk and used to access its internal API
+    :return: None
+    """
+    use_eu = None
+
+    if (debug) : print u'{} MessageType="ApteligentDebug"  Into setDataRegion'.format(DATETIME_OF_RUN)
+
+    try:
+        entities = entity.getEntities(['properties', 'crittercism','api'], namespace=myapp,
+                                        owner='nobody', sessionKey=sessionKey)
+        if (debug) : print u'{} MessageType="CritterDebug" entities are {}'.format(DATETIME_OF_RUN, entities)
+        use_eu = int(entities['dataCenterRegion'].value)
+        if (debug) : print u'{} MessageType="CritterDebug" use_eu is {}'.format(DATETIME_OF_RUN, use_eu)
+    except Exception, e:
+        print u'{} MessageType="ApteligentDebug" Could not get a data region because {} Defaulting to US'.format(DATETIME_OF_RUN, e)
```

Comment:
Some of these lines are really long.  Please use an 80 chr ruler.  https://www.python.org/dev/peps/pep-0008/#maximum-line-length